apps/root/src/pages/dca/create-position/index.tsx

```diff
@@ -21,8 +21,18 @@ const SwapContainer = ({ handleChangeNetwork }: SwapContainerProps) => {
   const dispatch = useAppDispatch();
   const currentNetwork = useCurrentNetwork();
   const { from: fromParam, to: toParam, chainId } = useParams<{ from: string; to: string; chainId: string }>();
-  const fromParamToken = useToken(fromParam, true, false, Number(chainId));
-  const toParamToken = useToken(toParam, true, false, Number(chainId));
+  const fromParamToken = useToken({
+    chainId: currentNetwork.chainId,
```

Comment:
shouldn't this be the Number(chainId)?

Reply:
We were not recognizing network names, this fixes that.
When the chainId param is `optimism`, its recognized and dispatched to the store. So we will admit urls that have /ethereum/custom-address

Reply:
There is no place here where it actually updates the currentNetwork (which btw refers to the wallet connected network)

Reply:
It was the selected network, imported as current network 🫠